logger/handlers/ringbuffer.go

```diff
@@ -0,0 +1,134 @@
+package handlers
```

Comment:
Why is this `handlers` (plural) instead of `handler`? I think we tend to stay with singular terms (`model`, not `models`, `encoder`, not `encoders`...)

I don't actually like the name `handler` here at all, since it isn't descriptive, and it seems at odds with the usage of `handler` in `net/http`. But I don't really care enough about that to make a big deal of it.